src/Prima/Pyxis/Form/Autocomplete.elm

```diff
@@ -80,6 +85,20 @@ type alias AutocompleteConfig model =
     List (AutocompleteOption model)
 
 
+{-| Internal. Filter alias.
+-}
+type alias Filter =
+    Maybe String
+
+
+{-| Internal. The `State` of the `AutocompleteChoice` list
+-}
+type ChoicesStatus
```

Comment:
Questo tipo può essere rappresentato (anche con un maggior controllo) dal tipo `RemoteData` che già usiamo in varie applicazioni (https://package.elm-lang.org/packages/krisajenkins/remotedata/latest/).

Reply:
ho detto io di non usarlo per evitare una dipendenza per introdurre semplicemente un tipo...